recipes/pyboat/meta.yaml

```
@@ -0,0 +1,73 @@
+{% set name = "pyboat" %}
+{% set version = "0.7.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 546c81d1cb952e25cd7660642741530e12c3b5053bacd54c7e745c4eae8becee
+
+build:
+  number: 0
```

I think it might be possible to make this noarch python
```suggestion
  number: 0
  noarch: python
```